apps/web/pages/api/book/confirm.ts

```diff
@@ -115,6 +116,21 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       return res.status(400).json({ message: "booking already confirmed" });
     }
 
+    /** If we are confirming a booking that need pay and does not have payment,
+     * it would not have to save in destination calendar */
```

Comment:
Probably a better wording could be:
`When a booking that requires payment its being confirmed but doesn't have any payment, we shouldn’t save it on DestinationCalendars`